### PR TITLE
Fix broken CreateOrUpdateRepoCustomPropertyValues

### LINKED ISSUE #3021: Bug in CreateOrUpdateRepoCustomPropertyValues?
We are looking to replace our custom-built methods for listing and updating custom property values, with the methods provided in v57.

It seems the CreateOrUpdateRepoCustomPropertyValues method has a bug however. We haven't actually tested it yet, but shouldn't the last argument be `properties []*CustomPropertyValue`?

```go
// Wrong type in last arg?
func (s *OrganizationsService) CreateOrUpdateRepoCustomPropertyValues(ctx context.Context, org string, repoNames []string, properties []*CustomProperty) (*Response, error) {
	u := fmt.Sprintf("orgs/%v/properties/values", org)

	params := struct {
		RepositoryNames []string          `json:"repository_names"`
		Properties      []*CustomProperty `json:"properties"`
	}{
		RepositoryNames: repoNames,
		Properties:      properties,
	}

	req, err := s.client.NewRequest("PATCH", u, params)
	if err != nil {
		return nil, err
	}

	return s.client.Do(ctx, req, nil)
}
```

### PR DESCRIPTION
**Pull request by peter-aglen**
Closes #3021.

### CODE REVIEW

**Review by gmlewis — APPROVED** [review]
Thank you, @peter-aglen !
LGTM.

Awaiting second LGTM+Approval from any other contributor to this repo before merging.

**Review by liaodaniel — APPROVED** [review]
LGTM

**Comment by gmlewis** [comment]
Thank you, @liaodaniel !
Merging.

**Pull request merged**
